docs/02-app/01-building-your-application/06-optimizing/10-third-party-libraries.mdx

```diff
@@ -240,7 +240,7 @@ docs](https://developers.google.com/maps/documentation/embed/embedding-map).
 | `allowfullscreen` | Optional | Property to allow certain map parts to go full screen.                                              |
 | `loading`         | Optional | Defaults to lazy. Consider changing if you know your embed will be above the fold.                  |
 | `q`               | Optional | Defines map marker location. _This may be required depending on the map mode_.                      |
-| `center`          | Optional | Defines the center of the map view.                                                                     |
```

Comment:
prettier fix corrected this whitespace at the end of the line, which is unrelated to this PR but I couldn't find any additional prettier-check errors.